<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCE5FF;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:7pt; color:#1f1e1e;&quot;&gt;(8 characters long including a number, a special character and an uppercase letter)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="3">
    <spacer name="horizontalSpacer_16">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_8">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:7pt; color:#1f1e1e;&quot;&gt;(20 characters long with multiple numbers, special characters and uppercase letters)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:8pt; color:#333333;&quot;&gt;Generate a Standard Strength Password&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_7">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:8pt; color:#333333;&quot;&gt;Generate a Super Strong Password&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="btn_3">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF8000; border-radius: 5px; color: #FFF; font-size: 18px; border: 1px solid #FF0000;  margin: 0px 10px 0px 10px</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_1">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF8000; border-radius: 5px; color: #FFF; font-size: 18px; border: 1px solid #FF0000;  margin: 0px 10px 0px 10px</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:14pt; color:#333333;&quot;&gt;Generate an Extra Strong Password&lt;/span&gt;&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="btn_2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF8000; border-radius: 5px; color: #FFF; font-size: 18px; border: 1px solid #FF0000;  margin: 0px 10px 0px 10px</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="horizontalSpacer_12">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="txt_1">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF; border-radius: 5px; border: 0px solid; margin: 0px 10px;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="1" colspan="2">
    <spacer name="horizontalSpacer_14">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid; margin: 0px 10px 0px 10px</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'trebuchet ms','sans-serif'; font-size:7pt; color:#1f1e1e;&quot;&gt;(12 characters long with multiple numbers, special characters and uppercase letters)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QTextEdit" name="txt_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF; border-radius: 5px; border: 0px solid; margin: 0px 10px;</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="QTextEdit" name="txt_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF; border-radius: 5px; border: 0px solid; margin: 0px 10px;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
